src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements Drawable, AreaCalculable {
+    private double sideA;
+    private double sideB;
+    private double height;
+
+    public IsoscelesTrapezoid(double sideA, double sideB, double height, String color) {
+        super.setColor(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.height = height;
+    }
+
+    @Override
+    public double area() {
+        return (sideA + sideB) * height / 2;
```

Comment:
The method name 'area' should be a verb, such as 'calculateArea', to clearly represent the action it performs.